app/controllers/api/v1/reservations_controller.rb

```diff
@@ -0,0 +1,40 @@
+class Api::V1::ReservationsController < ApplicationController
+  # before_action :set_reservation, only: %i[show destroy]
+
+  def index
+    @reservations = Reservation.joins(:doctor).pluck(
+      'reservations.id, doctors.name, reserve_date, city'
+    )
+    render json: @reservations
+  end
+
+   def create
+    @reservation = Reservation.new(reservation_params)
+    # @reservations.user = current_user
```

Comment:
- [ ]  Kindly remove this old piece of commented code to improve the cleanliness of your repo. 👍